src/components/ContentPreview/ContentPreview.scss

```diff
@@ -30,6 +30,10 @@
         right: 0;
     }
 
+    .bcpr-navigate-left {
```

Comment:
Do we need to match the border radius similar to the navigate right?

Reply:
I should just remove that, not sure whats its doing